src/arrays/referencing_array_and_its_elements/medium/q11/VowelNames.java

```diff
@@ -0,0 +1,29 @@
+package arrays.referencing_array_and_its_elements.medium.q11;
+
+/*
+Write a Java program that asks the user for 6 names and stores them in an array of Strings. Then, count how many of the
```

Comment:
Change this question so that the user has to print out all the name ending with the letter 'b' instead of counting them. Also taking a larger input size.